orbitize/gaia.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+from astroquery.gaia import Gaia
+from astropy import units as u
+
+class GaiaLogProb(object):
+    """
+    Class to compute the log probability of an orbit with respect to a single 
```

Comment:
Add a comment somewhere that we are currently only fitting for the position of the star from Gaia, not the proper motion of the Gaia epoch?